addons/im_livechat/controllers/chatbot.py

```diff
@@ -14,7 +14,7 @@ def chatbot_restart(self, channel_id, chatbot_script_id):
         chatbot = request.env['chatbot.script'].browse(chatbot_script_id)
         if not discuss_channel or not chatbot.exists():
             return None
-        chatbot_language = self._get_chatbot_language()
+        chatbot_language = request.env["chatbot.script"]._get_chatbot_language()
```

Comment:
`_get_chatbot_language ` could become alias for `request.env["chatbot.script"]. _get_chatbot_language()`, to avoid duplicate code in case we need to fix both LOCs at the same time.